src/Fist/Http/src/Request.php

```diff
@@ -0,0 +1,255 @@
+<?php
+
+namespace Fist\Http;
+
+class Request
```

Comment:
__[phpmd]__
* The class Request has an overall complexity of 50 which is very high. The configured complexity threshold is 50.

<!-- Comment by [SideCI](https://sideci.com) -->